### DISCUSSION TITLE
目前正在v1.0公测中，这是可用版本，欢迎大家使用、贡献、交流~

### DISCUSSION THREAD

**Discussion by yyc-git** [Meta3D-Technology/Meta3D#43] [Announcements]
Meta3D目前正在v1.0公测中，这是第一个真正可用的版本，欢迎大家在这里交流，可以给出您使用的感受、建议、反馈等，感谢大家~

[进入Meta3D平台](https://meta3d-production-5eol5gce9a6b9c-1302358347.tcloudbaseapp.com/)
[进入官网](https://meta3d-website.4everland.app/)

**Comment by yyc-git** [comment]
建议1：改进Scene View的缺失提示可以改进下

描述：我创建了个空画布后就点开始运行，会报错

**Reply by yyc-git** [reply]
已改进

**Comment by WuKunhuan** [comment]
<img width="939" alt="Screenshot 2023-12-20 at 12 56 21" src="https://github.com/Meta3D-Technology/Meta3D/assets/79775708/e104ebdd-c5ca-4017-9554-c20aabaf25d9">

如何创建这里面的side panels？

**Reply by yyc-git** [reply]
点击左侧Scene Tree中的节点（如Cube），就会显示Inspector

**Reply by yyc-git** [reply]
已修复

现在会显示默认的Inspector

**Comment by yyc-git** [comment]
Bug: 输入框输入字母，出现NaN
![image](https://img2023.cnblogs.com/blog/419321/202312/419321-20231220133150903-1255194797.png)

**Reply by yyc-git** [reply]
已修复

**Comment by yyc-git** [comment]
Bug: 编辑器：Inspector看不到，需要点击Scene Tree的节点后才显示Inspector

**Reply by yyc-git** [reply]
已修复

现在会显示默认的Inspector

**Comment by yyc-git** [comment]
问题：编辑器能否上传模型？
答：可以上传glb模型
![image](https://img2023.cnblogs.com/blog/419321/202312/419321-20231220133805148-872686075.png)
点击红框，选择glb

**Comment by l1305599284** [comment]
模型上传后，点击旋转的+-按钮，旋转轴不正确

**Reply by yyc-git** [reply]
已修复

解决方案：
action 保存euler

**Comment by yyc-git** [comment]
Bug: 编辑器：导出单一事件： 导出为空文件

**Reply by yyc-git** [reply]
已修复

修复方法：
编辑器加入ImportSingleEvent Action

**Comment by yyc-git** [comment]
Bug: 新手引导最后一步 点 Finish 没反应

**Reply by yyc-git** [reply]
已修复

**Comment by yyc-git** [comment]
1.运行“完整的编辑器”
2.加载超大glb模型（800M）

报错：
offset is out of bounds

**Reply by yyc-git** [reply]
解决方案
增大editor-whole包中的配置数据：geometryCount

不过目前已经占了800M内存了，不宜再增加内存

以后可让用户自定义该配置数据
目前先不修改

**Comment by yyc-git** [comment]
运行编辑器报错：window.open(...) is null

**Reply by yyc-git** [reply]
已修复

解决方案：
catch 报错，弹一个提示：请允许浏览器弹窗

**Comment by zhuifengzl** [comment]
有没有使用教程，小白看不太懂
